src/document.rs

```diff
@@ -226,6 +227,24 @@ impl<'a> Document<'a> {
 
         Ok(self)
     }
+
+    pub fn get_line_num(node: &Node) -> HowserResult<usize> {
+        let parent = node.capabilities
+            .traverse
+            .as_ref()
+            .ok_or(HowserError::CapabilityError)?
+            .parent()?;
+        let line_num = node.capabilities
+            .get
+            .as_ref()
+            .ok_or(HowserError::CapabilityError)?
+            .get_start_line()?;
+
+        match (parent, line_num) {
+            (Some(parent), 0) => Document::get_line_num(&parent),
```

Comment:
What does it mean to get 0 back from get_start_line()?

Reply:
It means that you have an inline node or some other nested node that cmark doesn't feel like reporting an actual line number for.

Reply:
So the logic is, if I don't seem to have a line number, perhaps my parent does...

Reply:
:cerealspit: ok, this seems like a good thing to comment about.

Reply:
Commented